source/ways-of-working/oat/index.html.md.erb

```diff
@@ -0,0 +1,189 @@
+  * Links to High/Low Level Design Documents or a high level purpose of each application.
+  * Details of Azure Infrastructure being used like keyvaults, Databases.
+  * Dependency of any external services.
+
+### Service Level Agreements
+
+This OAT is to capture any specific NFRs unique to the service/product that needs to be captured and evidenced.
+
+- Service team should identify and document SLAs of the applicable components in the OAT task.
+- Any external SLAs other than Azure should be explicitly listed in SOG.
+- Below are SLA details for some standard Azure services.
+
+| Azure service | SLA |
+| - | - |
+| Azure Kubernetes Service | [Kubernetes](https://azure.microsoft.com/en-us/support/legal/sla/kubernetes-service/v1_1/) |
+| Azure Container Registry | [Container-registry](https://azure.microsoft.com/en-us/support/legal/sla/container-registry/v1_1/) |
+| Azure Application Gateway | [Application-gateway](https://azure.microsoft.com/en-us/support/legal/sla/application-gateway/v1_2/) |
+| Azure Front Door | [Frontdoor](https://azure.microsoft.com/en-us/support/legal/sla/frontdoor/v1_0/) |
+| Azure Firewall | [Azure-firewall](https://azure.microsoft.com/en-us/support/legal/sla/azure-firewall/v1_1/) |
+| Azure DNS | [DNS](https://azure.microsoft.com/en-us/support/legal/sla/dns/v1_1/) |
+| Azure Content Delivery Network | [CDN](https://azure.microsoft.com/en-us/support/legal/sla/cdn/v1_0/) |
+| Azure Storage Account | [Storage Account](https://azure.microsoft.com/en-us/support/legal/sla/storage/v1_5/) |
+| Azure Active Directory | [Azure AD](https://azure.microsoft.com/en-us/support/legal/sla/active-directory/v1_0/) |
+| Azure DevOps | [Azure Devops](https://azure.microsoft.com/en-us/support/legal/sla/azure-devops/v2_0/) |
+| Azure PostgreSQL Database | [PostgreSQL](https://azure.microsoft.com/en-us/support/legal/sla/postgresql/v1_1/) |
+| Azure Service Bus | [Service Bus](https://azure.microsoft.com/en-us/support/legal/sla/service-bus/v1_1/)|
+
+### Licensing
+
+This OAT is to ensure the software and licenses for infrastructure and hosting is appropriately licensed and supported for the proposed design of the environments required for the service.
+
+- Development team to assess and document in SOG if there are any tools/ dependencies which are not covered under standard open source licensing.
+- Tools like Camunda, Launch Darkly can be marked as "Platform Managed"
+
+### Support Capability - Incident Handling
+
+This OAT is to ensure there is a robust support model for the product as part of the delivery.
+
+- SOG should be updated with support model along with details about `Assignment Group` in Service Now.
+- Standard support model is documented in [confluence](https://tools.hmcts.net/confluence/display/DIP/Public+Beta+Support)
+
+### Error Handling Framework
+
+This OAT is to ensure facilities are in place for errors/exceptions to recorded to be referred against in case of production issues.
```

Comment:
still reads a bit funny
```suggestion
This OAT is to ensure facilities are in place for errors / exceptions to recorded that can be referred to in case of production issues.
```